# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
# pyForceFieldConstraints test fails on some machines:
# Default number of steps for ff.Minimize is 200 and the default force tolerance is 0.0001.
# The minimize does not converge within 200 steps and therefore we increse the number of steps to 500.
diff -ru rdkit-Release_2024_03_5/Code/ForceField/Wrap/testConstraints.py rdkit-Release_2024_03_5_relax_pyForceFieldConstraints_test/Code/ForceField/Wrap/testConstraints.py
--- rdkit-Release_2024_03_5/Code/ForceField/Wrap/testConstraints.py	2024-07-19 09:46:29.000000000 +0200
+++ rdkit-Release_2024_03_5_relax_pyForceFieldConstraints_test/Code/ForceField/Wrap/testConstraints.py	2025-01-27 12:39:11.746639187 +0100
@@ -160,7 +160,12 @@
     ff = ChemicalForceFields.UFFGetMoleculeForceField(m)
     self.assertTrue(ff)
     ff.UFFAddAngleConstraint(1, 3, 6, False, -10.0, 10.0, 100.0)
-    r = ff.Minimize()
+    # THEMBL r = ff.Minimize()  
+    # Minimize((ForceField)self[, (int)maxIts=200[, (float)forceTol=0.0001[, (float)energyTol=1e-06]]]) → int :¶
+    # THEMBL default maxIts is 200 and forceTol is 0.0001.
+    # on some machines Minimize does not converge within 200 steps. Therefore we
+    # increase the number of steps.
+    r = ff.Minimize(500)
     self.assertTrue(r == 0)
     conf = m.GetConformer()
     angle = rdMolTransforms.GetAngleDeg(conf, 1, 3, 6)
